packages/app/src/types/verifiableCredentials/index.ts

```diff
@@ -10,17 +10,17 @@ export interface ICryptkeeperVerifiableCredential {
   metadata: IVerifiableCredentialMetadata;
 }
 
-export interface IAddVerifiableCredentialArgs {
+export interface IAddVCArgs {
   serializedVerifiableCredential: string;
   verifiableCredentialName: string;
 }
 
-export interface IRenameVerifiableCredentialArgs {
+export interface IRenameVCArgs {
   verifiableCredentialHash: string;
   newVerifiableCredentialName: string;
```

Comment:
```suggestion
  vcHash: string;
  newVCName: string;
```

Reply:
The same here, it can be just hash, newName.